### ISSUE TITLE
Review and update the service and deployment names

### ISSUE THREAD

**Issue by ssorj** [skupperproject/skupper#224]
For example, skupper-messaging - perhaps that should be called skupper-router now.

**Comment by ssorj** [comment]
```
~/code/skupper-example-hello-world$ kcp
NAME                                          READY   STATUS    RESTARTS   AGE
hello-world-frontend-699d4d458b-qt6kk         1/1     Running   0          139m
skupper-router-559c5fff5d-sbzzf               2/2     Running   0          139m
skupper-service-controller-5957446654-skt8f   1/1     Running   0          139m

~/code/skupper-example-hello-world$ kcd
NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
hello-world-frontend         1/1     1            1           139m
skupper-router               1/1     1            1           139m
skupper-service-controller   1/1     1            1           139m

~/code/skupper-example-hello-world$ kcs
NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)                           AGE
hello-world-backend    ClusterIP      10.106.9.124    <none>          8080/TCP                          138m
hello-world-frontend   LoadBalancer   10.105.151.19   10.105.151.19   8080:30375/TCP                    138m
skupper-controller     LoadBalancer   10.103.218.45   10.103.218.45   8080:31931/TCP                    139m
skupper-internal       LoadBalancer   10.105.78.26    10.105.78.26    55671:31168/TCP,45671:31894/TCP   139m
skupper-messaging      ClusterIP      10.110.185.37   <none>          5671/TCP                          139m
```

In this output, what deployment, if any, does the skupper-controller service correspond to?

**Comment by grs** [comment]
`what deployment, if any, does the skupper-controller service correspond to?`

The skupper-service-controller. However the important question I think is what purpose does the service exits for and that is at present to provide access to the console, so probably 'console' or 'console-server' would be a better name.

**Comment by ssorj** [comment]
Recording the proposed changes:

service/skupper-messaging -> service/skupper-router
service/skupper-controller -> service/skupper-console

**Comment by ajssmith** [comment]
Addressed in 0.5.0 release
